addons/account/report/inherited_layouts.xml

```diff
@@ -6,7 +6,7 @@
     <xpath expr="//ul[@name='financial_infos']" position="inside">
         <t t-if="not company.custom_footer">
             <t t-set="is_first" t-value="True"/>
-            <t t-foreach="company.bank_journal_ids" t-as="journal">
+            <t t-foreach="company.bank_journal_ids.sudo()" t-as="journal">
```

Comment:
I don't agree that a record rule is specifically more secure, since it gives you access to all fields of the record. In a template, you are limited to the fields displayed. As long as these specific fields contain public information, it should be fine.

In this case, this is the footer template, so the information displayed here (bank name, account number and BIC number) is likely to be found on any printed document. I don't think this could cause a security issue to disclose these information.

However, in doubt, let's avoid the `sudo` as much as possible, I agree.